server/docker-entrypoint.sh

```diff
@@ -12,12 +13,104 @@ fi
+    pass=$(generate_user_or_password)
+    echo "######################################################################################"
+    echo "# Using domain mode but no management user and/or password provided.                 #"
+    echo "# Management user and password has been generated.                                   #"
+    echo "# Management user: $usr                                                        #"
+    echo "# Management password: $pass                                                   #"
+    echo "#                                                                                    #"
+    echo "# You can provide management user and password details via environment variables.    #"
+    echo "#                                                                                    #"
+    echo "# Docker run example:                                                                #"
+    echo "#     docker run ... -e \"MGMT_USER=user\" -e \"MGMT_PASS=changeme\" ...             #"
+    echo "#                                                                                    #"
+    echo "# Dockerfile example:                                                                #"
+    echo "#     ENV MGMT_USER admin                                                            #"
+    echo "#     ENV MGMT_PASS admin                                                            #"
+    echo "#                                                                                    #"
+    echo "# Kubernetes Example:                                                                #"
+    echo "#     spec:                                                                          #"
+    echo "#       containers:                                                                  #"
+    echo "#       - args:                                                                      #"
+    echo "#         image: jboss/infinispan-server:...                                         #"
+    echo "#         ...                                                                        #"
+    echo "#         env:                                                                       #"
+    echo "#         - name: MGMT_USER                                                          #"
+    echo "#           value: admin                                                             #"
+    echo "#         - name: MGMT_PASS                                                          #"
+    echo "#           value: admin                                                             #"
+    echo "#                                                                                    #"
+    echo "# OpenShift client example:                                                          #"
+    echo "#     oc new-app ... -e MGMT_USER=user -e MGMT_PASS=changeme ...                     #"
+    echo "######################################################################################"
```

Comment:
I'm fine either way. I thought it could be useful, but happy to remove it if its too noisy/big.